src/scripts/farming/Farming.ts

```diff
@@ -971,7 +1163,10 @@ class Farming implements Feature {
             15000,
             20,
             [10, 30, 0, 0, 30],
+            40,
             BerryColor.Blue,
+            7.5,
+            BerryFirmness.Very_Hard,
```

Comment:
Apicot is listed as only Hard in your link

Reply:
Serebii and its gen 3 berry tag says Hard, so this will be changed.